articles/applications/reference/wildcard-subdomains.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Wildcard for Subdomains
+description: Learn how using a wildcard for subdomains can function in application configuration.
+toc: true
+topics:
+  - applications
+contentType: reference
+useCase:
+  - build-an-app
+---
+# Wildcard for Subdomains
+
+Several fields in [Application Settings](/dashboard/reference/settings-application) allow for use of wildcards for subdomains for convenient registration of URLs:
+ 
+* **Allowed Callback URLs**: Set of URLs to which Auth0 is allowed to redirect users after they authenticate.
+* **Allowed Logout URLs**: List of URLs to which you can redirect users after they log out from Auth0.
+* **Allowed Origins (CORS)**: Set of URLs that will be allowed to make requests from JavaScript to Auth0 API (typically used with CORS).
+
+See [Application Settings Best Practices](/best-practices/application-settings) for recommended settings for applications, which suggests to avoid using wildcards for subdomains in in application callbacks and allowed origins. If you choose to use them, you can use the star symbol (`*`) as a wildcard for subdomains, but it must be used in accordance with the following rules in order to properly function.
```

Comment:
```suggestion
See [Application Settings Best Practices](/best-practices/application-settings) for recommended settings for applications, which suggests to avoid using wildcards for subdomains in application callbacks and allowed origins. If you choose to use them, you can use the star symbol (`*`) as a wildcard for subdomains, but it must be used in accordance with the following rules in order to properly function.
```